caravel/forms.py

```diff
@@ -430,6 +430,18 @@ def __init__(self, viz):
             'compare_suffix': TextField(
                 'Comparison suffix',
                 description="Suffix to apply after the percentage display"),
+            'table_timestamp_format': FreeFormSelectField(
+                'Table Timestamp Format',
+                default='smart_date',
+                choices=[
```

Comment:
Let's not copy pasta. Can you refactor the choices for this and x_axis_timeformat in a CONSTANT at the top of the module?